Content.Server/Construction/ConstructionSystem.Graph.cs

```diff
@@ -324,11 +324,20 @@ public bool ChangeNode(EntityUid uid, EntityUid? userUid, string id, bool perfor
                 }
             }
 
-            // We set the graph and node accordingly.
-            ChangeGraph(newUid, userUid, construction.Graph, construction.Node, false, newConstruction);
+            // TODO: Is this call necessary?
+            UpdatePathfinding(newUid, newConstruction);
```

Comment:
this isn't necessary since both ChangeNode and SetPathfindingTarget call UpdatePathfinding

Reply:
I've removed this call now. Sorry it took a while - didn't get a GitHub email for some reason.